f8a_worker/workers/bookkeeper.py

```diff
@@ -0,0 +1,21 @@
+import os
+from selinon import StoragePool
+from f8a_worker.base import BaseTask
+
+
+class BookkeeperTask(BaseTask):
+    description = 'Keep bookkeeping data on RDS'
+    # we don't want to add `_audit` etc into the manifest submitted
+    add_audit_info = False
+
+    def execute(self, arguments):
+        self._strict_assert(arguments.get('external_request_id'))
+        self._strict_assert(arguments.get('data'))
+
+        self.log.info("Request id = %s" % arguments.get('external_request_id'))
+
+        postgres = StoragePool.get_connected_storage('BayesianPostgres')
```

Comment:
I would rather suggest to create a new adapter besides `BayesianPostgres` and place the storing logic there. Main benefits:

1. We do not mix analyses storing logic with the bookkeeping logic
2. If we decide to introduce a new RDS instance just for data like this, we will just pass different connection string (will help us scale).
3. It will more fit the current design where we configure tasks using YAML config files.

Anyway 1. is more reasonable in short term.